src/steps/os/linux.rs

```diff
@@ -59,7 +59,9 @@ impl Distribution {
             Some("centos") | Some("rhel") | Some("ol") => Distribution::CentOS,
             Some("clear-linux-os") => Distribution::ClearLinux,
             Some("fedora") => {
-                return if let Some(variant) = variant {
+                return if section.get("OSTREE_VERSION").is_some() {
+                    Ok(Distribution::FedoraImmutable)
```

Comment:
Would you mind add a comment here to state the reason for this `OSTREE_VERSION` check?